enterprise/cli/provisionerdaemonstart.go

```diff
@@ -72,6 +75,48 @@ func (r *RootCmd) provisionerDaemonStart() *serpent.Command {
 			interruptCtx, interruptCancel := inv.SignalNotifyContext(ctx, agpl.InterruptSignals...)
 			defer interruptCancel()
 
+			logOpts := []clilog.Option{
+				clilog.WithFilter(logFilter...),
+				clilog.WithHuman(logHuman),
+				clilog.WithJSON(logJSON),
+				clilog.WithStackdriver(logStackdriver),
+			}
+			if verbose {
+				logOpts = append(logOpts, clilog.WithVerbose())
+			}
+
+			logger, closeLogger, err := clilog.New(logOpts...).Build(inv)
+			if err != nil {
+				// Fall back to a basic logger
+				logger = slog.Make(sloghuman.Sink(inv.Stderr))
+				logger.Error(ctx, "failed to initialize logger", slog.Error(err))
+			} else {
+				defer closeLogger()
+			}
+
+			// Wait for coderd liveness, with retry.
+			retryCtx, retryCancel := context.WithTimeout(ctx, time.Hour)
+			defer retryCancel()
+			for retrier := retry.New(100*time.Millisecond, 10*time.Second); retrier.Wait(retryCtx); {
+				lcCtx, lcCancel := context.WithTimeout(inv.Context(), time.Second*30)
```

Comment:
Shouldn't this use the retryCtx?

```suggestion
				lcCtx, lcCancel := context.WithTimeout(retryCtx, time.Second*30)
```